spec/rubocop/cop/style/fetch_env_var_spec.rb

```diff
@@ -89,6 +89,34 @@
     it 'does not register an offense with `||`' do
       expect_no_offenses(<<~RUBY)
         ENV['X'] ||= y
+        x ||= ENV['X'] ||= y
+      RUBY
+    end
+  end
+
+  context 'when the node is a receiver of `&&=`' do
+    it 'does not register an offense with `&&`' do
```

Comment:
It may be redundant (The same is true for L89).

```suggestion
    it 'does not register an offense' do
```